MCProd/tasks.py

```diff
@@ -12,9 +12,13 @@
 from .run_prod import run_prod, step_to_file_name
 from .mk_l1tuple import mk_l1tuple
 from RunKit.grid_helper_tasks import CreateVomsProxy
-from RunKit.sh_tools import timed_call_wrapper, update_kerberos_ticket
+from RunKit.run_tools import timed_call_wrapper, update_kerberos_ticket
+from RunKit.grid_tools import gfal_copy_safe, gfal_exists
+import sys
+sys.path.append('/usr/lib64/python3.9/site-packages')
```

Comment:
is it needed?

Reply:
No, it's a remnant of previous attempts. I removed it in the latest commit.